src/mixins/java/org/spongepowered/common/mixin/inventory/event/world/inventory/AbstractContainerMenuMixin_Inventory.java

```diff
@@ -162,6 +162,14 @@ public abstract class AbstractContainerMenuMixin_Inventory implements TrackedCon
 
     }
 
+    @Inject(method = "broadcastFullState", at = @At("HEAD"))
+    private void impl$broadcastFullStateWithTransactions(final CallbackInfo ci) {
+        if (!PhaseTracker.SERVER.onSidedThread()) {
+            return;
+        }
+        this.bridge$detectAndSendChanges(false, false);
```

Comment:
```suggestion
        this.bridge$detectAndSendChanges(false, false);
        this.impl$broadcastDataSlots();
        this.shadow$sendAllDataToRemote();
        this.impl$captureSuccess = true; // Detect mod overrides
        ci.cancel();
```

Reply:
`broadcastFullState` calls `triggerSlotListeners`, `updateDataSlotListeners` and `sendAllDataToRemote` but your change calls  `synchronizeDataSlotToRemote`, `updateDataSlotListeners` and `sendAllDataToRemote` which is not the same, why ?

Reply:
>  but I didn't understand for sendSlotChange.

At the moment we are sending individual slot changes in the `bridge$detectAndSendChanges` (L237) even when the `synchronize` is set to false which I suggested to fix. Then after this and the `impl$broadcastDataSlots` fix the methods are performing identical logic to vanilla.

Reply:
Oh I didn't see that `slotChanged` was called in `bridge$detectAndSendChanges`.
I will make the changes !

Reply:
Done !